### ISSUE TITLE
[Bug Report][2.4.9] Editable Stepper Header Not Keyboard Accessible 

### ISSUE THREAD

**Issue by flpehr0** [vuetifyjs/vuetify#14022]
### Environment
**Vuetify Version:** 2.4.9
**Vue Version:** 2.6.12
**Browsers:** Chrome 92.0.4515.107, Google Chrome, Mozilla Firefox
**OS:** Windows 10

### Steps to reproduce
When on a page with a stepper that includes headers using the editable prop, attempt to use the keyboard to focus on and execute that step.

### Expected Behavior
keyboard focus should be moved to the actionable element, in this case the stepper header, and be executable with the keyboard. All focus styles that apply when clicked should apply when focused with the keyboard. 

### Actual Behavior
The keyboard skips over the clickable header entirely and is therefore not executable with the keyboard 

### Reproduction Link
<a href="https://codepen.io/flpehr0/pen/yLbqZJe" target="_blank">https://codepen.io/flpehr0/pen/yLbqZJe</a>

### Other comments
Adding a tabindex="0" when editable should be a good first step as well as ensuring that it will execute with a keyboard and not only a mouse.
